### DISCUSSION TITLE
关于开启ssl证书的一些配置，小白求问

### DISCUSSION THREAD

**Discussion by cashy14** [hhyo/Archery#2497] [General]
各位大佬有配置过ssl证书吗，按照文档的方式配置好了，日志也显示nginx启动，但是docker ps 没有看到nginx的容器，按照文档上的配置是直接用https://宿主机:加上监听端口号就好了嘛

**Comment by LeoQuote** [comment]
建议别动 archery 的任何配置, 使用反向代理比如 nginx 来加 ssl 证书

**Reply by cashy14** [reply]
这个要怎么配置，在外部部署nginx 无法加载静态资源，登陆会报错
